Makefile

```
@@ -157,12 +157,18 @@ fmt: ## Check formatting of go sources.
 	@$(MAKE) go.init
 	@$(MAKE) go.fmt
 
+.PHONY: markdownlint
+# the markdownlint  target requires
+# markdownlint-cli2 to be available locally in the PATH
+markdownlint: ## Check formatting of documentation sources
+	@markdownlint-cli2 "Documentation/**/**.md" "#Documentation/Helm-Charts/**" --config .markdownlint-cli2.cjs ; \
```

Unless I'm interpreting things incorrectly, I don't think the semicolon is necessary. Also, yamllint and pylint don't hide their invocation with `@`, so let's keep the same behavior here to keep consistency.

```suggestion
	markdownlint-cli2 "Documentation/**/**.md" "#Documentation/Helm-Charts/**" --config .markdownlint-cli2.cjs
```